docker.md

```diff
@@ -506,14 +500,14 @@ $ docker build -t hello:v3 .
 We should be now able to run the new image version without supply additional arguments:
```

Comment:
We should now be able to run the new image version without supplying additional arguments (typo - rewording)